core/src/lib.rs

```diff
@@ -264,9 +259,9 @@ pub trait ProjectAll<Parent, F> {
 /// }
 /// ```
 ///
-/// But it is better to just derive `Field` on the types that you need to, and
-/// then use the [`chain`](trait.Fields.html#method.chain) combinator to project
-/// to the fields of fields
+/// Best practice is to derive `Field` on the needed types, using the
+/// [`chain`](trait.Fields.html#method.chain) combinator to project to the
+/// fields of `Field` rather than implementing `Field` manually
```

Comment:
Oh, this is so much better than I was expecting!

Reply:
From what I've read so far, it looks great!